README.md

```diff
@@ -1 +1,32 @@
-# review-project
\ No newline at end of file
+## Review past project
+In this project, we build the middle section of the mobile website:
```

Comment:
Please update the Readme.md file

Reply:
- [ ] This comment is vague, kindly be clearer when giving a comment. e.g. The README's "Getting Started" sections are currently empty. Why don't you add content, and how to run your project, to them so your README is more informative and professional?

Reply:
Kindly update the Readme.md file and make it adaptable to your project.